authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,45 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
```

Comment:
Great work Asava. However, you can consider eliminating this space for consistency

Reply:
Looks good Asava.